pages/_profiles/Dataset/0.6-DRAFT.html

```diff
@@ -0,0 +1,393 @@
+  - Organization
+  - Person
+  marginality: Recommended
+  property: creator
+  type: ''
+  type_url: ''
+- bsc_description: ''
+  cardinality: ONE
+  controlled_vocab: ''
+  description: ' A downloadable form of this dataset, at a specific location, in a
+    specific format.'
+  example: ''
+  expected_types:
+  - DataDownload
+  marginality: Recommended
+  property: distribution
+  type: ''
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: ' A data catalog which contains this dataset. Supersedes [includedDataCatalog](https://schema.org/includedDataCatalog),
+    [catalog](https://schema.org/catalog).
+
+    Inverse property: [dataset](https://schema.org/dataset)'
+  example: ''
+  expected_types:
+  - DataCatalog
+  marginality: Recommended
+  property: includedInDataCatalog
+  type: ''
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: "Use to link a Dataset to the Study that it was generated from. A resource\
+    \ that was used in the creation of this resource. This term can be repeated for\
+    \ multiple sources. For example, http://example.com/great-multiplication-intro.html.\
+    \ \nSupersedes isBasedOnUrl."
+  example: ''
+  expected_types:
+  - Creativework
+  - Product
+  - URL
+  marginality: Recommended
+  property: isBasedOn
+  type: ''
+  type_url: ''
+- bsc_description: ''
+  cardinality: MANY
+  controlled_vocab: ''
+  description: ' A technique or technology used in a [Dataset](https://schema.org/Dataset)
+    (or [DataDownload](https://schema.org/DataDownload), [DataCatalog](https://schema.org/DataCatalog)),
+    corresponding to the method used for measuring the corresponding variable(s) (described
+    using [variableMeasured](https://schema.org/variableMeasured)). This is oriented
+    towards scientific and scholarly dataset publication but may have broader applicability;
+    it is not intended as a full representation of measurement, but rather as a high
+    level summary for dataset discovery.
+
+
+    For example, if [variableMeasured](https://schema.org/variableMeasured) is: molecule
+    concentration, [measurementTechnique](https://schema.org/measurementTechnique)
+    could be: "mass spectrometry" or "nmr spectroscopy" or "colorimetry" or "immunofluorescence".
+
+
+    If the [variableMeasured](https://schema.org/variableMeasured) is "depression
+    rating", the [measurementTechnique](https://schema.org/measurementTechnique) could
+    be "Zung Scale" or "HAM-D" or "Beck Depression Inventory".
+
+
+    If there are several [variableMeasured](https://schema.org/variableMeasured) properties
+    recorded for some given data object, use a [PropertyValue](https://schema.org/PropertyValue)
+    for each [variableMeasured](https://schema.org/variableMeasured) and attach the
+    corresponding [measurementTechnique](https://schema.org/measurementTechnique).'
+  example: ''
+  expected_types:
+  - DefinedTerm
```

Comment:
DefinedTerm is not part of the range in schema.org, see https://schema.org/measurementTechnique
I know it would be useful and it is under discussion by the community but I do not think we have got to any agreement yet so I would wait to include it. @ivanmicetic @albangaignard @AlasdairGray any thoughts?